src/infoDetails/infoElement.component.test.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import { createShallow, createMount } from '@material-ui/core/test-utils';
+import InfoElement from './infoElement.component';
+import { harderAttribute } from '../malcolmWidgets/table/table.stories';
+
+const state = {
+  malcolm: {
+    blocks: {
+      test1: {
+        attributes: [harderAttribute],
+      },
+    },
+  },
+};
+describe('InfoElement', () => {
+  let shallow;
+  let mount;
+  let mockStore;
+  beforeEach(() => {
+    harderAttribute.calculated.testStuff = {
+      somethingFalse: false,
+      somethingTrue: true,
+      aString: 'testing!',
+      aNumber: 1,
+    };
+    mockStore = configureStore();
+    shallow = createShallow({ dive: true });
+    mount = createMount();
+  });
+
+  it('renders correctly with value given', () => {
+    const wrapper = shallow(
+      <InfoElement
+        key={1}
+        label="Test element"
+        value="testing..."
+        alarm={null}
+        tag="info:multiline"
+        store={mockStore(state)}
+      />
+    );
+    expect(wrapper.dive()).toMatchSnapshot();
+  });
+
+  it('renders correctly with paths given', () => {
+    const wrapper = shallow(
+      <InfoElement
+        key={1}
```

Comment:
We might want to revisit this later, it seems like the info element relies on a lot of properties